### PR TITLE
add Coop-panel event

### PR DESCRIPTION
**Pull request by KarolinaXin**

### CODE REVIEW

**Comment by evykassirer** [comment]
(waiting on room details)

**Comment by bdamerac** [comment]
We booked MC 2065

**Comment by annalorimer** [comment]
@Bhargavi1997 @KarolinaXin y'all ok to update the commit with the room information?

File: content/W2017/Coop-panel.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: Winter 2017 Co-op Panel
+Date: 2017-01-19
+Category: Events
+Tags: social, panels, coop
+Slug: co-op-panel-w17
+Author: Karolina Xin
+Summary: Ask questions about Co-op and job applications
+
+Nervous about Co-op? Or got a ton of questions? Well then come to the Co-op Panel, where you
```
Comment (evykassirer) [inline content/W2017/Coop-panel.md]: make this line less than 80 char

**Comment by annalorimer** [comment]
The name of the file should be "W17-event-coop-panel.md" instead of "Coop-panel.md". You can look at the directories from other terms to see the file naming conventions :)

**Comment by bdamerac** [comment]
Can you add details about the people on panel?
They're as follows:

Anna Lorimer -- 3A Computer Science & Combinatorics and Optimization
 Dejero (QA, W15), Audiobooks.com (Web Developer, F15), IBI Intelligent Systems (Junior Software     Developer, S16), Arctic Wolf Networks (Software Developer, W17)

Arshia Mufti -- 3A Computer Science
 WeMesh (W16), Bazaarvoice (F16)

Shirley Miao (3B Computer Science)
 Google (S15), Facebook (F15), Stripe (S16)

Amy Qiu (2B Computer Science)
 Microsoft Explorer Intern (S16)

**Comment by evykassirer** [comment]
@Bhargavi1997 I updated your comment with a few more details/changes

**Comment by rtkalorimer** [comment]
My bad, that should be IBI Intelligent Systems (instead of System).

**Comment by annalorimer** [comment]
I'm gonna merge this because it needs to get up asap. We can go back and add stuff later.

**Comment by annalorimer** [comment]
Merging because this needs to go up asap. @KarolinaXin and I will work on adding the panelists later :)

**Comment by annalorimer** [comment]
Merging because this needs to get up asap. @KarolinaXin and I will add the panelists retroactively :)

**Comment by annalorimer** [comment]
Merging because this needs to get up asap. @KarolinaXin and I will add the panelists retroactively :)

**Comment by evykassirer** [comment]
did you deploy the site?

**Pull request merged**